src/patternfly/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,217 @@
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: var(--pf-c-skeleton__element--after--ZIndex);
+    display: block;
+    width: 100%;
+    height: 100%;
+    content: "";
+    background: linear-gradient(var(--pf-c-skeleton__element--after--LinearGradientAngle), var(--pf-c-skeleton__element--after--LinearGradientColorStop1), var(--pf-c-skeleton__element--after--LinearGradientColorStop2), var(--pf-c-skeleton__element--after--LinearGradientColorStop3));
+    transform: translateX(var(--pf-c-skeleton__element--after--TranslateX));
+    animation: var(--pf-c-skeleton__element--after--AnimationName) var(--pf-c-skeleton__element--after--AnimationDuration) var(--pf-c-skeleton__element--after--AnimationTimingFunction) var(--pf-c-skeleton__element--after--AnimationDelay) var(--pf-c-skeleton__element--after--AnimationIterationCount);
+  }
+
+  // Circle
+  &.pf-m-circle {
+    --pf-c-skeleton__element--BorderRadius: var(--pf-c-skeleton__element--m-circle--BorderRadius);
+  }
+
+  // Width
+  &.pf-m-width-sm {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-sm--Width);
+
+    height: var(--pf-c-skeleton__element--m-width-sm--Width);
+  }
+
+  &.pf-m-width-md {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-md--Width);
+
+    height: var(--pf-c-skeleton__element--m-width-md--Width);
+  }
+
+  &.pf-m-width-lg {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-lg--Width);
+
+    height: var(--pf-c-skeleton__element--m-width-lg--Width);
+  }
+
+  &.pf-m-height-sm {
+    --pf-c-skeleton__element--Height: var(--pf-c-skeleton__element--m-height-sm--Height);
+  }
+
+  &.pf-m-height-md {
+    --pf-c-skeleton__element--Height: var(--pf-c-skeleton__element--m-height-md--Height);
+  }
+
+  &.pf-m-height-lg {
+    --pf-c-skeleton__element--Height: var(--pf-c-skeleton__element--m-height-lg--Height);
+  }
+
+  // Width modifiers
+  &.pf-m-width-10 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-10--Width);
+  }
+
+  &.pf-m-width-15 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-15--Width);
+  }
+
+  &.pf-m-width-20 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-20--Width);
+  }
+
+  &.pf-m-width-25 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-25--Width);
+  }
+
+  &.pf-m-width-30 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-30--Width);
+  }
+
+  &.pf-m-width-35 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-35--Width);
+  }
+
+  &.pf-m-width-40 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-40--Width);
+  }
+
+  &.pf-m-width-45 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-45--Width);
+  }
+
+  &.pf-m-width-50 {
+    --pf-c-skeleton__element--Width: var(--pf-c-skeleton__element--m-width-50--Width);
+  }
+
```

Comment:
It's kind of odd we don't do +5% increments after 50%. We could also use the same widths that are in the sizing utility - https://github.com/patternfly/patternfly/blob/61898c01f7dcbb7937d9ba624ab374166f9d96cb/src/patternfly/utilities/Sizing/sizing.scss#L5-L14

Reply:
Yeah I was following what was in the table. But Ill follow the sizing utility instead and maybe one day we can update the table